test/ocsp/helper/helper.go

```diff
@@ -236,6 +250,11 @@ func parseAndPrint(respBytes []byte, cert, issuer *x509.Certificate, expectStatu
 	if resp.Status != expectStatus {
 		return nil, fmt.Errorf("wrong CertStatus %d, expected %d", resp.Status, expectStatus)
 	}
+	if isFlagSet("expect-reason") {
+		if resp.RevocationReason != *expectReason {
```

Comment:
I'd combine this with the if statement on the line above.

Reply:
Done